LICENSE

```diff
@@ -2162,3 +2162,37 @@ The externally maintained libraries used by Node.js are:
     NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
     SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
   """
+
+- glob(7), located at src/node_path,cc, is licensed as follows:
+  """
+    Valid-License-Identifier: MIT
+    SPDX-URL: https://spdx.org/licenses/MIT.html
+    Usage-Guide:
+      To use the MIT License put the following SPDX tag/value pair into a
+      comment according to the placement guidelines in the licensing rules
+      documentation:
+        SPDX-License-Identifier: MIT
+    License-Text:
+
+    MIT License
+
+    Copyright (c) <year> <copyright holders>
```

Comment:
This line seems incorrect to me. I'm not sure if the license builder needs work or something - just pointing it out.

Reply:
this is just the content of a file pulled from the internet